api/v1/app.py

```diff
@@ -11,12 +11,13 @@
 
 app = Flask(__name__)
 app.register_blueprint(app_views)
+""" CORS for Task 13 """
 cors = CORS(app, resources={r"/*": {"origins": "0.0.0.0"}})
 
 
 @app.errorhandler(404)
 def invalid_route(e):
-    """ 404 """
+    """ 404 for Task 6? """
     return (jsonify({"error": "Not found"}), 404)
```

Comment:
I don't think you need the parenthesis around jsonify and 404
but then again I don't know if that affects it or not